server/utils/utils.ts

```diff
@@ -69,7 +69,7 @@ export const formatAssessmentData = (
   assessment: AssessmentResponse,
   areas: AssessmentAreaConfig[],
 ): AssessmentAreas => {
-  if (assessment === null) {
+  if (assessment === null || assessment === undefined || assessment.assessment === undefined) {
```

Comment:
```suggestion
  if (!assessment.assessment) {
```
this could be written like this, right?